### ISSUE TITLE
Error on macos: Project file not found at

### ISSUE THREAD

**Issue by clarity99** [hvanbakel/CsprojToVs2017#161]
It seems that the different slashes confuse it:

```
csproj-to-2017 ./cryptoportfolio.sln 
Solution parsing started.
Project found: cryptoportfolio\cryptoportfolio.csproj
Project file not found at: /Users/rob/Dropbox/src/cryptoportfolio/cryptoportfolio\cryptoportfolio.csproj

```
Project file is there, but it should use path /Users/rob/Dropbox/src/cryptoportfolio/cryptoportfolio/cryptoportfolio.csproj

In the SLN the proj is defined as 
`Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cryptoportfolio", "cryptoportfolio\cryptoportfolio.csproj", "{3C579E5C-ABF9-4286-AAF3-8230FB37BEB7}"
`

**Comment by hvanbakel** [comment]
@clarity99 would you mind verifying the 2.0.0-rc1 for this?

**Comment by andrew-boyarshin** [comment]
@hvanbakel I'm afraid it does not have the said fix. You only fixed that for `ProjectReference`s, @clarity99 was having issues with .sln file, it is handled in `SolutionReader`. #170 correctly fixes that in both places, but testing is needed indeed (I don't have running Linux/macOS at this moment).

**Comment by andrew-boyarshin** [comment]
@clarity99 fix has made it into 2.0.0-rc2, feel free to try it out.

**Comment by clarity99** [comment]
Thanks, I'll try it out over the weekend and report
